bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/ResumingStreamingResultScanner.java

```diff
@@ -46,6 +51,14 @@
   private Context operationContext;
   private Context rpcContext;
 
+  private RetryOptions retryOptions;
+
+  // The number of times we've retried after a timeout
+  private AtomicInteger timeoutRetryCount = null;
```

Comment:
I'm not certain, but it doesn't look like this class is accessed from multiple threads. If that's true, then I think we should just mark this class as @NotThreadSafe